R/g.part4.R

```diff
@@ -678,13 +677,13 @@ g.part4 = function(datadir = c(), metadatadir = c(), f0 = f0, f1 = f1,
                   }
                   delta_t1 = diff(as.numeric(spocum.t$end))
                   spocum.t$dur = correct01010pattern(spocum.t$dur)
-                  
+
```

Comment:
Can you please use Ctrl+A and then Ctrl+I indentation, because that logically follows the indentation of the code. Line 680 starts right under the 's' of 'spocumt$dur' on the previous line and not all the way on the left. It is not practical to be using different indentation styles and the one I propose is easiest as it is just two short-keys.

Reply:
Thanks, I always do Ctrl+A and then Ctrl+I, so this was weird. Then I observed that the indentation was removed from my empty lines after saving the file, and this is produced by an option in RStudio. To get the indentation in the empty lines, both _Tools/Project options_ and _Tools/Global options_ should have the _Strip trailing horizontal whitespace_ when saving box **unchecked**.

Reply:
Oh, that explains it. So in theory anyone who makes a pull request could have this set in a way that is inconsistent with how I was doing it. Well, let's at least try to keep it consistent between ourselves as I guess it is also annoying for you if I would change it back every time.